src/lib/Microsoft.Health.Events/Telemetry/Exceptions/EventHubExceptionTelemetryProcessor.cs

```diff
@@ -23,26 +23,69 @@ public static class EventHubExceptionTelemetryProcessor
             EnsureArg.IsNotNull(exception, nameof(exception));
             EnsureArg.IsNotNull(logger, nameof(logger));
 
-            logger.LogError(exception);
+            var ex = CustomizeException(exception);
+
+            logger.LogError(ex);
 
             if (shouldLogMetric)
```

Comment:
I think we can just get rid of `shouldLogMetric` as an input since we are processing all event hub exceptions within this class and can decide in here if certain exceptions should not be logged (although for now I don't see why we shouldn't log a metric for these cases)

Reply:
Sounds good. I was wondering the same but forgot to ask.